src/acp/policy.ts

```diff
@@ -228,6 +251,272 @@ export function getDenyModes(policy: Policy): AccessModes {
   };
 }
 
+/**
+ * ```{note} There is no Access Control Policies specification yet. As such, this
+ * function is still experimental and subject to change, even in a non-major release.
+ * ```
+ *
+ * Initialise a new, empty [[ResourcePolicy]] for the given Resource.
+ *
+ * @param resourceWithAcr The Resource to which the Policy is to apply.
+ * @param name The name that identifies this Policy.
+ */
+export function createResourcePolicyFor(
+  resourceWithAcr: WithAccessibleAcr,
+  name: string
+): ResourcePolicy {
+  const acr = internal_getAcr(resourceWithAcr);
+  const url = new URL(getSourceUrl(acr));
+  url.hash = `#${name}`;
```

Comment:
```suggestion
  url.hash = name;
```
Is specifying the `#` mandatory ? When i run node locally, I get the following
```
> const url = new URL("https://test.example/resource")
undefined
> url.href
'https://test.example/resource'
> url.hash = "fragment"
'fragment'
> url.href
'https://test.example/resource#fragment'
```

Reply:
I don't think it's mandatory for setting, but since [it's always present when reading](https://developer.mozilla.org/en-US/docs/Web/API/URL/hash), I figured it might be best to be consistent with that when writing.